src/components/ArchiveSidebar.svelte

```diff
@@ -0,0 +1,85 @@
+<script>
+  import { fly } from "svelte/transition";
+  import {
+    Col,
+    Container,
+    Row,
+    Card,
+    CardHeader,
+    CardBody,
+    Icon,
+  } from "sveltestrap/src";
+  import { stores } from "@sapper/app";
+
+  import ViewCard from "./ViewCard.svelte";
+
+  const { session } = stores();
+
+  export let id;
+  export let is_archive_sidebar_shown;
+  export function closeSidebar() {
+    is_archive_sidebar_shown = false;
+  }
+</script>
+
+{#if is_archive_sidebar_shown}
+  <nav class="sidebar" transition:fly={{ x: 350 }}>
+    <Card>
+      <CardHeader>
+        <Container>
+          <Row>
+            <Col class="sidebar-sidebar-header" xs="10">Archived Cards</Col>
+            <Col xs="1"
+              ><button class="borderless-button" on:click={closeSidebar}
+                >x</button
+              >
```

Comment:
Remove the orphan `>`s. I think this happens when the formatter incorrectly breaks long lines. The fix is to manually newline at the ending `>`, which the formatter will try to respect.

This should become:

```js
<Col xs="1">
  <button class="borderless-button" on:click={closeSidebar}>
    x
  </button>
```